org.eclipse.xsmp.forms/META-INF/MANIFEST.MF

```diff
@@ -33,7 +33,8 @@ Require-Bundle: org.eclipse.xsmp,
  org.eclipse.xsmp.ui,
```

Comment:
Is it still required to depend on xsmp.ui ?

Reply:
Fixed